bridge/pkg/terra/sender.go

```diff
@@ -0,0 +1,102 @@
+package terra
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/certusone/wormhole/bridge/pkg/terra/terra.go/client"
```

Comment:
Why is `terra.go` copied to the repository? Will it be maintained here in the external terra.go repository?